### DISCUSSION TITLE
fail to build ring in Windows

### DISCUSSION THREAD

**Discussion by cherish-ltt** [briansmith/ring#2140] [General]
```The` following warnings were emitted during compilation:

warning: ring@0.17.8: cc.exe: error: unrecognized debug output level 'full'
warning: ring@0.17.8: cc.exe: error: unrecognized command-line option '-mmacosx-version-min=10.7'
warning: ring@0.17.8: cc.exe: error: unrecognized command-line option '-arch'

error: failed to run custom build command for ring v0.17.8 ```

use  cargo build --target=x86_64-apple-darwin in Windows10

**Comment by briansmith** [comment]
Cross-compilation from a Windows host to an Apple target isn't well supported yet, unfortunately. I hope to improve this in a future release. There are issues tracking this.

**Reply by cherish-ltt** [reply]
I solved this problem using `zigbuild` [cargo-zigbuild](https://crates.io/crates/cargo-zigbuild).
thank you!
